django_lti_tool_provider/views.py

```diff
@@ -146,7 +148,9 @@ def process_authenticated_lti(cls, request):
                 _logger.exception(u"Invalid LTI Request")
                 return HttpResponseBadRequest(u"Invalid LTI Request: " + e.message)
 
-        lti_data = cls._store_lti_parameters(request.user, lti_parameters)
+        lti_data = LtiUserData.store_lti_parameters(
```

Comment:
@Kelketek `store_lti_parameters` does not catch possible `WrongUserError`, so if it occurs here it would result in 500, which is semantically incorrect - it is caused by conflicting data, so 409 CONFLICT or 403 FORBIDDEN would make more sense.